docs/topics/spider-middleware.rst

```diff
@@ -328,6 +328,68 @@ Default: ``True``
 
 Whether to enable referer middleware.
 
+.. setting:: REFERER_POLICY
+
+REFERER_POLICY
+^^^^^^^^^^^^^^
+
+.. versionadded:: 1.4
+
+Default: ``'scrapy.spidermiddlewares.referer.DefaultReferrerPolicy'``
+
+.. reqmeta:: referrer_policy
```

Comment:
I'm ok with changing to `REFERRER_POLICY`. (It's a shame about that spelling mistake in the RFCs.)